doc/configuration.md

```diff
@@ -9,7 +9,6 @@ This document explains how to configure Pravega
     * [NFS](tier2.md#use-NFS-as-Tier2)
     * [Google Filestore Storage](tier2.md#use-google-filestore-storage-as-tier-2)
 * [Tune Pravega Configuration](pravega-options.md)
-* [Tune Bookkeeper Configuration](bookkeeper-options.md)
```

Comment:
This change does **not** look correct.
Skipping bookkeeper configuration altogether from Pravega configuration is not right.
As discussed, please move this page along with all linked pages into pravgea-operator wiki.

Reply:
Yes, now I have added it back and have added a wiki page for all the options in the configuration.md file and attached the corresponding wiki page to it.